main.go

```diff
@@ -13,16 +12,24 @@ import (
 	"os"
 	"path/filepath"
 	"strings"
+
+	"github.com/dave/dst"
+	"github.com/dave/dst/decorator"
 )
 
-const godocCommentFormat = "// %s missing godoc"
+var godocCommentFormat = "// %s missing godoc"
+var godocAllowNoSpace = false
```

Comment:
Do we really need to make a flag for this? Can we just allow comments without space between the // and the type name by default as it is a valid godoc.

Reply:
not really, added it not to change the default behavior, but will remove